services/movie_session.py

```diff
@@ -0,0 +1,52 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    session = MovieSession.objects.get(id=session_id)
```

Comment:
Try to update session only when optional parameters has been passed:
1. Create a dictionary.
2. Check if any of the optional arguments are passed to this function. In case if there are - you will add them to the dict.
3. In the end check if your dict has some values to update and if yes you will call update on queryset, and if there are not any you don't do anything